pxtlib/markdownMetadataParser.ts

```diff
@@ -12,7 +12,7 @@ namespace pxt {
     }
 
     export function getSectionsFromMarkdownMetadata(text: string): MarkdownSection[] {
-        const lines = text.split("\n");
+        const lines = text.split(/\r?\n/);
```

Comment:
for ref, this caused issues parsing text when it was saved and not yet committed in windows so that's why I suggested this simple fix, seems fine to me~